pkg/synopsysctl/cmd_create.go

```diff
@@ -713,6 +719,27 @@ func updatePolarisSpecWithFlags(cmd *cobra.Command, namespace string) (*polaris.
 	return &polarisSpec, nil
 }
 
+func polarisPostgresCheck(flagset *pflag.FlagSet) error {
+	if flagset.Lookup("postgres-container").Changed {
+		if flagset.Lookup("postgres-host").Changed || flagset.Lookup("postgres-port").Changed {
+			return fmt.Errorf("cannot change the host and port when using postgres container")
+		}
+		if flagset.Lookup("postgres-ssl-mode").Changed {
+			return fmt.Errorf("cannot enable SSL when using postgres container")
+		}
+	} else {
+		// External DB. Host and port are mandatory
+		cobra.MarkFlagRequired(flagset, "postgres-host")
```

Comment:
Since there is a default value for host and port, whether it will force to provide the values?

Reply:
Yes, I can confirm that it works and that the field is required even if there is a default value.